### DISCUSSION TITLE
Swiper only loads first 4 images of slider

### DISCUSSION THREAD

**Discussion by networkiok-1icetea** [nolimits4web/swiper#7392] [Q&A]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://www.davelo.ca/en/22-rockhopper-29.html

### Bug description

Swiper slides that include more than 4 images do not load correctly. Images 5+ are stuck loading as per the screenshot below. This happens on all product pages of our website. 

I have tested increasing the slidesPerView param to include 5 or 10 images and they all load "correctly' as seen in the second screenshot, but once we're back to 1 or 1.5 perView the bug comes back.

![image](https://github.com/nolimits4web/swiper/assets/89235409/81656728-44dd-400c-8b12-026319b5d314)
![image](https://github.com/nolimits4web/swiper/assets/89235409/64d4d431-94ad-4d08-ab38-f8f8c97aff46)

Here is the js, where I left all the possible fixes I've tried so far:

```$(document).ready(function() {
  var thisSwiper = [];
  $('.swiper-product-images').each(function(i) {
    var this_ID = "swiper-images";
    thisSwiper[i] = new Swiper(".swiper-product-images", {
      observer: true,
      observeParents: true,
      observeSlideChildren: true,
      watchSlidesProgress: true,
      watchSlidesVisibility: true,
      slidesPerGroup:1,
      slidesPerView:1,
      spaceBetween:0,
      speed:1000,
      breakpoints: {
        640: {
          slidesPerView: 1,
          spaceBetween: 0,
        },
        768: {
          slidesPerView: 1,
          spaceBetween: 0,
        },
        1024: {
          slidesPerView: 1.5,
          slidesPerGroup:1,
          spaceBetween: 5,
        },
      },
      pagination: {
        el:  '.swiper-pagination',
        clickable:true
      },
    });
  });
});```

### Expected Behavior

_No response_

### Actual Behavior

_No response_

### Swiper version

8.0.6

### Platform/Target and Browser Versions

OS: MacOS, Windows, Android, Broswer: Chrome, IE, Safari

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by AFSDARIF** [comment]
For the above solution I have created a config for each section.
![code-swipperjs](https://github.com/user-attachments/assets/5655f136-fdca-4e6e-a23e-794eb4402fb2)
